test/powershell/Language/Parser/TypeAccelerator.Tests.ps1

```diff
@@ -413,13 +413,13 @@ Describe "Type accelerators" -Tags "CI" {
             param($Accelerator, $Type)
             $TypeAcceleratorsType::Get[$Accelerator] | Should -Be ($Type)
         }
-    
+
         It 'Should have a type accelerator for non-dotnet-core type: <Accelerator>' -Skip:$IsCoreCLR -TestCases $extraFullPSAcceleratorTestCases {
```

Comment:
We should enable the test on Windows and correct `$totalAccelerators = 95` above.

Reply:
Missed the -Skip part of this test.  Will fix.  Not sure what you mean by "correct $totalAcclerators = 95 above" as 95 is the correct number.

Reply:
Seems we add 5 accelerators on Windows.